client/blocks/visit-site/index.jsx

```diff
@@ -22,7 +22,7 @@ export default function VisitSite( { siteSlug } ) {
 		return null;
 	}
 
-	const siteLink = <a href={ site.URL }>{ site.name.trim() }</a>;
+	const siteLink = <a href={ site.URL }>{ siteSlug }</a>;
```

Comment:
If you test with a site that's installed in a subdirectory, `siteSlug` will contain an escaped version of the forward slash. We need to unescape those and get them back to forward slashes:

![Screenshot 2022-11-21 at 12 26 08](https://user-images.githubusercontent.com/8436925/203027125-82074d51-d6c6-406c-8be1-7302e8c47a47.png)

Reply:
Did not know about that, cool. :-) Fixed in https://github.com/Automattic/wp-calypso/pull/70228/commits/4c39ded6c8b06b4fa58ba47edb5e1ed6723d56df